CHANGELOG.md

```diff
@@ -10,6 +10,16 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 [Unreleased][Unreleased_log]
 --------------
 ### Added
+- `oe_set_host_log_level(oe_log_level_t log_level)` allows applications to dynamically modify the verbosity of logs captured from host.
+  - This API only modifies the verbosity of logs generated by the host. This does not affect enclave log level verbosity.
+  - It can be called with or without logging callback set and is thread safe. There is no limit on number of calls per application.
+
+- `oe_set_enclave_log_level(oe_enclave_t* enclave, oe_log_level_t log_level)` is an host API that allows applications to dynamically modify the verbosity of logs generated by enclaves.
+  - This API is enabled only when logging.edl is imported.
+  - This API can be used functionally only after enclave creation and before enclave termination for an enclave. An application can call this API for multiple enclaves. There is no limit on number of calls.
+  - When using ocalls to log enclave logs, if the application needs to modify the verbosity of the enclave logs to be higher than current verbosity of host logs, the host log level verbosity also needs to be modified (using `oe_set_host_log_level`). Eg. When the current host and enclave log_level = ERROR, and would like to make the enclave_log_level = WARN, then both host and enclave log levels need to be set to WARN.
```

Comment:
Instead of having this behavior, can the API just return error if the target level is less than host log level?

Reply:
The API now returns OE_CONSTRAINT_FAILED in this scenario.